pennylane/debugging.py

```diff
@@ -241,13 +366,53 @@ def state():
 
 
 def expval(op):
-    """Compute the expectation value of an observable.
+    """Compute the expectation value of an observable at the
```

Comment:
Should we add a "see also" qml.expval?

Reply:
I think it might cause more confusion, since they are used in very different contexts.